### DISCUSSION TITLE
"Preparing wheel metadata" is killed while trying to install cfclient

### DISCUSSION THREAD

**Discussion by 0furkanyildiz** [bitcraze/discussions#588] [Q&A] [answered]
Hey there,
I'm trying to install cfclient using the virtual environment in my Ubuntu 20.04 system as explained in another discussion. However, I got stuck while trying to prepare wheel metadata. After nearly 3-4 minutes, this step is killed and therefore, the installation cannot be successful. (I have pip-installed wheel) I should say that I'm a beginner in any crazyflie content and trying to follow along instructions online and learn about it. I would be happy to receive any kind of suggestions.

![Screenshot from 2023-02-23 23-22-48](https://user-images.githubusercontent.com/103690610/221022891-eccb97fa-d442-4616-bde6-ad55156c3ef2.png)

**Comment by krichardsson** [comment, marked as answer]
Maybe it is related to #545
Please try to run `pip3 install --upgrade pip` first

**Reply by 0furkanyildiz** [reply]
Thank you so much, it worked!
